docs/modules/plugins/pages/plugin-mobile-app.adoc

```diff
@@ -136,3 +136,7 @@ Given I start mobile application with capabilities:
 |bstack:options.networkLogs     |true |
 |appium:options.clearSystemFiles|false|
 ----
+
+include::partial$generic-ui-steps.adoc[]
+
+include::partial$mobile-app-locators.adoc[]
```

Comment:
on the other hand if newbie opens the documentation and see hundreds of lines at the very beginning dedicated for locators explanations he will not understand why he needs all this stuff because he didn't reach steps section yet, I mean that installation, props and steps sections are more important, whereas locator is just a parameter

Reply:
сс @valfirst